packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -31,3 +31,13 @@ export const userLogPayloadGuard = z.object({
 });
 
 export type UserLogPayload = z.infer<typeof userLogPayloadGuard>;
+
+export const applicationLevelSettingsGuard = z.object({
+  applicationId: z.object({ application_skip_get_started: z.boolean() }),
+});
+
+export const adminConsoleConfigGuard = z.object({
+  application_skip_get_started: applicationLevelSettingsGuard.array(),
```

Comment:
do we need an array to store the value? Or maybe just a boolean field `show_get_started_on_app_management` that may applies to all the well created applications.

Reply:
the settings might vary by different applications?

Reply:
nah, i would say it's a global settings

Reply:
i would go with one single boolean